assets/xml/objects/object_yabusame_point.xml

```diff
@@ -1,6 +1,8 @@
 ﻿<Root>
     <File Name="object_yabusame_point" Segment="6">
-        <!-- <Blob Name="object_yabusame_point_Blob_000000" Size="0xD80" Offset="0x0" /> -->
-        <DList Name="object_yabusame_point_DL_000DC0" Offset="0xDC0" />
+        <Texture Name="gYabusamePoint30Tex" OutName="30_points" Format="ia8" Width="48" Height="24" Offset="0x0"/>
+        <Texture Name="gYabusamePoint60Tex" OutName="60_points" Format="ia8" Width="48" Height="24" Offset="0x480"/>
+        <Texture Name="gYabusamePoint100Tex" OutName="100_points" Format="ia8" Width="48" Height="24" Offset="0x900"/>
```

Comment:
```suggestion
        <Texture Name="gYabusamePoint30Tex" OutName="points_30" Format="ia8" Width="48" Height="24" Offset="0x0"/>
        <Texture Name="gYabusamePoint60Tex" OutName="points_60" Format="ia8" Width="48" Height="24" Offset="0x480"/>
        <Texture Name="gYabusamePoint100Tex" OutName="points_100" Format="ia8" Width="48" Height="24" Offset="0x900"/>
```
Would this be better? I think it's bad practice to start a filename with a number

Reply:
Oh right, forgot about windows